stdlib/random.ml

```diff
@@ -113,6 +113,8 @@ module State = struct
           (String.get_int64_le d2 0)
           (String.get_int64_le d2 8)
 
+  let full_init = reinit
```

Comment:
It doesn't look like `full_init` has the right type in the mli. (`reinit` takes two arguments, but the mli says `full_init` should have one argument: `val full_init : int array -> unit`.) Is this PR missing something?

Reply:
Never mind, I see this is `State.full_init`. This makes sense now.